models/match.py

```diff
@@ -95,6 +95,17 @@ class Match(ndb.Model):
     push_sent = ndb.BooleanProperty()  # has an upcoming match notification been sent for this match? None counts as False
     tiebreak_match_key = ndb.KeyProperty(kind='Match')  # Points to a match that was played to tiebreak this one
 
+    event_key = ndb.ComputedProperty(lambda self: self.event_key_name)
```

Comment:
These all shouldn't need to be ComputedProperties since you're building the separate search index. `@property` should work fine.